### PR TITLE
Add python skin update script

### PR DESCRIPTION
**Pull request by Lejo1**
The current Ingame downloader isn't able to download the whole database without spamming http errors.

So I just created a new download script in python which does this without any problems.
It's just a nice alternative to the Ingame downloader

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +30,12 @@ This Minetest mod offers changeable player skins with a graphical interface for
 
 You might want to run `minetest` in a Terminal/Console window to check the log output instantly.
 
+#### Python Download script
+
+Only requires >=python3 and requests `pip3 install requests`
```
Comment (SmallJoker) [inline README.md:35]:
```suggestion
**Requirements:**

 * Python 3
 * `requests` library: `pip3 install requests`  
 
```

File: updater/update_skins.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        file.write(name + "\n" + author + "\n" + license + "\n")
+        file.close()
+        print("Added #%s Name: %s Author: %s License: %s" % (id, name, author, license))
+        count += 1
+
+    else:
+        print("Fetching of preview of skin " + id + "failed!")
+
+
+print("Fetched " + str(count) + " skins!")
```
Comment (SmallJoker) [inline updater/update_skins.py:44]: `r = requests.get(` does the fetching.
This text is supposed to say:
```suggestion
print("Downloaded " + str(count) + " skins!")
```

File: updater/update_skins.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        file.close()
+        # Meta file
+        file = open("../meta/character_" + id + ".txt", "w")
+        file.write(name + "\n" + author + "\n" + license + "\n")
+        file.close()
+        print("Added #%s Name: %s Author: %s License: %s" % (id, name, author, license))
+        count += 1
+
+    else:
+        print("Fetching of preview of skin " + id + "failed!")
```
Comment (SmallJoker) [inline updater/update_skins.py:41]: `id + ".png"` is not the preview image.
```suggestion
        print("Failed to download skin id=" + id + ")
```

**Comment by Lejo1** [comment]
Changed that and created a bug report #49

**Comment by SmallJoker** [comment]
@bell07 If you don't have any objections, I'd like to merge this PR in the next few days.

**Comment by bell07** [comment]
No objections. The script seems clean and smart. Maybe we should close the #43 in favour of this script by the way.

**Pull request merged**